src/ops/fuelup_self.rs

```diff
@@ -18,6 +22,38 @@ pub fn attempt_install_self(download_cfg: DownloadCfg, dst: &Path) -> Result<()>
     Ok(())
 }
 
+const GOODBYE_HEADER: &str = "Thanks for hacking in Sway!";
+const GOODBYE_INFO: &str = "This will uninstall all Sway toolchains and data, and remove, $HOME/.fuelup/bin from your PATH environment variable.";
```

Comment:
I don't see where we're modifying the PATH. We should give users a separate choice about modifying the PATH if they don't use `--force`

Reply:
What do you mean? If we tell users that PATH is going to be modified, we should actually do it.

Reply:
Thanks @crodas Are we able to write a test for the code added in https://github.com/FuelLabs/fuelup/commit/d8c49245f3d5117fd41028e2a0755661783cce20